src/main/java/org/folio/domain/Condition.java

```diff
@@ -21,7 +23,10 @@
   ),
 
   MAX_NUMBER_OF_LOST_ITEMS("72b67965-5b73-4840-bc0b-be8f3f6e047e",
-    (summary, limit) -> summary.getNumberOfLostItems() >= limit.getValue()
+    (summary, limit) -> summary.getOpenLoans().stream()
+      .map(OpenLoan::getItemLost)
+      .filter(BooleanUtils::isTrue)
```

Comment:
.filter(OpenLoan::getItemLost) ?

Reply:
`itemLost` is a nullable `Boolean`. Passing it as predicate directly into a `filter()` may cause NPE.

Reply:
but as I can see you do like this in org.folio.rest.handlers.FeeFineBalanceChangedEventHandler#removeLoanIfLastLostItemFeeWasClosed. Is that a mistake or null cannot be there?

Reply:
Added default values for both flags in OpenLoan to avoid NPE.